### DISCUSSION TITLE
block unknown client

### DISCUSSION THREAD

**Discussion by gab12** [AdguardTeam/AdGuardHome#2964] [Q&A]
Hello,
I often see that unknown connections (CN, RU etc.) are querying and use my Adguard Home.
I have to check regularly and block them.
All my devices have a specific client id.
Those who do not are therefore necessarily intruders.

I don't know how they find my adguard, but I would like a feature to block all requests from an unknown client. but impossible. How will you do it if possible? thank you in advance

**Comment by ameshkov** [comment]
Hi you can use DNS settings -> Access settings -> Allowed clients to allow only your clients.

**Reply by gab12** [reply]
Thank you for your answer.
The problem is that it is not a checkbox .. you must indicate the client's IP and not his ID client.
And on my 4G phone, my ip is not fixed. :( Si I can't whitelisted him.

note: still today i still had 5 news unknown clients. (21 requests :( )

**Reply by ameshkov** [reply]
Please note, that access settings currently only work for plain non-encrypted DNS.

You can even block ALL clients like this `0.0.0.0/0`, but your DOH resolver will continue to work.
